content/en/docs/concepts/policy/limit-range.md

```diff
@@ -37,28 +37,28 @@ A limit range is enforced in a particular namespace when there is a
 
 - The administrator creates one `LimitRange` in one namespace.
 - Users create resources like Pods, Containers, and PersistentVolumeClaims in the namespace.
-- The `LimitRanger` admission controller enforces defaults  limits for all Pods and Container that do not set compute resource requirements and tracks usage to ensure it does not exceed resource minimum , maximum and ratio defined in any  `LimitRange` present in the namespace.
-- If creating or updating a resource (Pod, Container, PersistentVolumeClaim) violates a limit range  constraint, the request to the API server will fail with HTTP status code `403 FORBIDDEN` and a message explaining the constraint that would have been violated.
-- If limit range is activated in a namespace for compute resources like `cpu` and `memory`, users must specify
-  requests or limits for those values; otherwise, the system may reject pod creation.
-- LimitRange validations occurs only at Pod Admission stage, not on Running pods.
+- The `LimitRanger` admission controller enforces defaults and limits for all Pods and Container that do not set compute resource requirements and tracks usage to ensure it does not exceed resource minimum, maximum and ratio defined in any `LimitRange` present in the namespace.
+- If creating or updating a resource (Pod, Container, PersistentVolumeClaim) that violates a limit range  constraint, the request to the API server will fail with an HTTP status code `403 FORBIDDEN` and a message explaining the constraint that have been violated.
+- If a limit range is activated in a namespace for compute resources like `cpu` and `memory`, users must specify
+  requests or limits for those values. Otherwise, the system may reject pod creation.
+- Limit Range validations occurs only at Pod Admission stage, not on Running pods.
 
 
 Examples of policies that could be created using limit range are:
 
-- In a 2 node cluster with a capacity of 8 GiB RAM, and 16 cores, constrain Pods in a namespace to request 100m and not exceeds 500m for CPU , request 200Mi and not exceed 600Mi
```

Comment:
The portion of the example here for `request 200Mi and not exceed 600Mi` did not make sense so I removed it. Is it referring to memory specifications for the limit range?

Reply:
> request 200Mi and not exceed 600Mi

I take that to mean that Pods will have a 200MiB memory request and to require that matched Pods have a memory limit at or below 600MiB.

Reply:
Thanks! Let me add it back/make it more clear.